app/controllers/collection_drafts_controller.rb

```diff
@@ -74,9 +74,7 @@ def update
 
       if gkr_enabled? && params[:recommended_keywords_viewed] == 'true'
         # create keyword_recommendation
-        get_resource.record_recommendation_provided
-
-        # TODO: if/else and logging
+        get_resource.record_recommendation_provided(params[:gkr_request_id], params[:recommended_keyword_list])
```

Comment:
So the recommendations that are stored are what get logged, and these are the recommendations that are in the hidden input field. The recommendations in the hidden input are after they are 'reconciled', so after removing what was already stored in the draft previously. Is that what we want to log, or do we want to log all of the recommendations that were received?

Reply:
I think we want to log the pre-reconciled list.

Reply:
I stored the pre-reconciled list in the form (:recommended_keyword_list), so that is what is being logged here.

Reply:
`hidden_field_tag("recommended_keyword_list", keyword_recommendation_list(@keyword_recommendations))`
is being set: `@keyword_recommendations = reconcile_recommendations(keyword_recommendations)`
so the list is the reconciled list.

Reply:
Yes, that is incorrect.

Reply:
Changed the code to use the original keyword_recommendations in the hiden_field_tag.